ospfd/ospf_te.c

```diff
@@ -3347,8 +3415,12 @@ static uint16_t show_vty_link_subtlv_max_rsv_bw(struct vty *vty,
 	fval = ntohf(top->value);
 
 	if (vty != NULL)
-		vty_out(vty, "  Maximum Reservable Bandwidth: %g (Bytes/sec)\n",
-			fval);
+		if (!json)
+			vty_out(vty, "  Maximum Reservable Bandwidth: %g (Bytes/sec)\n",
+				fval);
+		else
+			json_object_double_add(json, "maxReservableBandwidth",
```

Comment:
nit: `maximumReservableBandwidth`, as we have `maximumBandwidth` (or use maxXX for both).

Reply:
Done